app/src/views/Cart.vue

```diff
@@ -129,11 +129,13 @@
         </div>
       </div>
 
-      <div class="py-8 border-b border-grey-100" :class="{ hidden: !showEquivalentContributionAmount }">
-        <div class="flex gap-x-4 justify-end">
+      <div class="py-8 border-b border-grey-100" :class="{ hidden: hideEquivalentContributionAmount }">
+        <div v-if="equivalentContributionAmount" class="flex gap-x-4 justify-end">
```

Comment:
If `equivalentContributionAmount` was `0` then `hideEquivalentContributionAmount` would be `true` so `equivalentContributionAmount` here is always going to hold a value and I don't think that the spinner path would ever be followed?

Reply:
Ah cool, so this was an attempt to fix the loading state that was occurring in #233 - I personally never actually saw it, but @mds1 and @phutchins were seeing it. This was more of a head start to get it fixed. The ideas was that if it wasn't loaded yet, it would show the loading spinner. (see line 38 here for reference: https://github.com/dcgtc/dgrants/blob/main/app/src/views/Home.vue)

If #233 is resolved by #229 already, then we can discard `2ad124d` and keep `b6b892f` (code refactor).

Reply:
@mds1 since you experience #233, can you verify if it's resolved in main (or this PR)?

Reply:
Sure, I can try tomorrow! If your internet is too fast for you to see it (I'm jealous), you could always simulate slower speeds with the browser dev tools or just modify the component state to force the app into the loading state

Reply:
(also, if you don't put back ticks around the commit hashes, GitHub will link to them and format it with the code style anyway, which is handy, e.g. `b6b892f` vs b6b892f)

Reply:
So I think @gdixon is right here that we never hit that branch. I don't see this resolved in #229 or here, because it's not really an error but more of a UX thing. Here's a slowed down gif showing what happens. You'll notice that "Estimated matching amount" starts at zero until it's loaded, but should show a spinner instead. Similarly, "Equivalent to" is hidden until loaded, but should always be present and also show a spinner

![image](https://im.ezgif.com/tmp/ezgif-1-5f59b6868968.gif)

Reply:
thanks @mds1 - I'll take a look and simulate slower speeds to test

Reply:
Sorry if this was unclear @mk1r—two thoughts:
1. I think the loading spinner should be in place of the numbers, not the whole div, otherwise you get a jumpy page from different div sizes. See image below, where loading spinner is taking up a large portion of the page, and the "Equivalent to" text is gone, so you can't tell what's loading
2. Somewhere along the way, the top-level Cart.vue if-else logic broke, as the page-level loading spinner never shows. The whole page is blank while data is loading

![image](https://user-images.githubusercontent.com/17163988/134673530-bdd38fc9-6c50-488f-b7a2-8061c24e669c.png)